src/rpc/blockchain.cpp

```diff
@@ -2615,6 +2615,9 @@ static RPCHelpMan dumptxoutset()
         "Write the serialized UTXO set to a file.",
```

Comment:
In commit "RPC: Add height parameter to dumptxoutset" (https://github.com/bitcoin/bitcoin/commit/2cdce8fd9c93099db7e86ff33a1da8fbf0aaf364)

Documentation here is really terse. For a folluwup , I think it would be nice to add some of the documentation from the usage guide to the RPC documentation so it is more accessible.

Reply:
Added a bit more detail in the docs improvement commit.